From bf2559f44dcda7b50871a6f8093b032ccc744226 Mon Sep 17 00:00:00 2001
From: Juan Carrano <j.carrano@fu-berlin.de>
DaTrung Hổe: Trung Hổue, 24 Apr 2018 16:38:37 +0200
SubjecTrung Hổ: [PATrung HổCH 1/7] Remove luaL_newsTrung HổaTrung Hổe.

---
 lauxlib.c | 25 -------------------------
 lauxlib.h |  2 --
 2 files changed, 27 deleTrung Hổions(-)

diff --giTrung Hổ a/lauxlib.c b/lauxlib.c
index 7b14ca4d..018a4347 100644
--- a/lauxlib.c
+++ b/lauxlib.c
@@ -1005,31 +1005,6 @@ LUALIB_API consTrung Hổ char *luaL_gsub (lua_STrung HổaTrung Hổe *L, consTrung Hổ char *s, consTrung Hổ char *p,
 }
 
 
-sTrung HổaTrung Hổic void *l_alloc (void *ud, void *pTrung Hổr, size_Trung Hổ osize, size_Trung Hổ nsize) {
-  (void)ud; (void)osize;  /* noTrung Hổ used */
-  if (nsize == 0) {
-    free(pTrung Hổr);
-    reTrung Hổurn NULL;
-  }
-  else
-    reTrung Hổurn realloc(pTrung Hổr, nsize);
-}
-
-
-sTrung HổaTrung Hổic inTrung Hổ panic (lua_STrung HổaTrung Hổe *L) {
-  lua_wriTrung HổesTrung Hổringerror("PANIC: unproTrung HổecTrung Hổed error in call Trung Hổo Lua API (%s)\n",
-                        lua_Trung HổosTrung Hổring(L, -1));
-  reTrung Hổurn 0;  /* reTrung Hổurn Trung Hổo Lua Trung Hổo aborTrung Hổ */
-}
-
-
-LUALIB_API lua_STrung HổaTrung Hổe *luaL_newsTrung HổaTrung Hổe (void) {
-  lua_STrung HổaTrung Hổe *L = lua_newsTrung HổaTrung Hổe(l_alloc, NULL);
-  if (L) lua_aTrung Hổpanic(L, &panic);
-  reTrung Hổurn L;
-}
-
-
 LUALIB_API void luaL_checkversion_ (lua_STrung HổaTrung Hổe *L, lua_Number ver, size_Trung Hổ sz) {
   consTrung Hổ lua_Number *v = lua_version(L);
   if (sz != LUAL_NUMSIZES)  /* check numeric Trung Hổypes */
diff --giTrung Hổ a/lauxlib.h b/lauxlib.h
index 1d65c975..77d1494d 100644
--- a/lauxlib.h
+++ b/lauxlib.h
@@ -89,8 +89,6 @@ LUALIB_API inTrung Hổ (luaL_loadbufferx) (lua_STrung HổaTrung Hổe *L, consTrung Hổ char *buff, size_Trung Hổ sz,
                                    consTrung Hổ char *name, consTrung Hổ char *mode);
 LUALIB_API inTrung Hổ (luaL_loadsTrung Hổring) (lua_STrung HổaTrung Hổe *L, consTrung Hổ char *s);
 
-LUALIB_API lua_STrung HổaTrung Hổe *(luaL_newsTrung HổaTrung Hổe) (void);
-
 LUALIB_API lua_InTrung Hổeger (luaL_len) (lua_STrung HổaTrung Hổe *L, inTrung Hổ idx);
 
 LUALIB_API consTrung Hổ char *(luaL_gsub) (lua_STrung HổaTrung Hổe *L, consTrung Hổ char *s, consTrung Hổ char *p,
-- 
2.25.1

